pkg/default_validator_test.go

```diff
@@ -578,19 +579,21 @@ func TestDefaultValidator_legalEntityFromContract(t *testing.T) {
 	t.Run("Empty message returns error", func(t *testing.T) {
 		ctx := createContext(t)
 		defer ctx.ctrl.Finish()
-		_, err := ctx.v.legalEntityFromContract(SignedIrmaContract{IrmaContract: irma2.SignedMessage{}})
+		_, err := ctx.v.legalEntityFromContract(&SignedIrmaContract{IrmaContract: irma2.SignedMessage{}, ContractTemplate: &ContractTemplate{}})
 
-		assert.NotNil(t, err)
-		assert.True(t, errors.Is(err, ErrInvalidContractText))
+		if !assert.NotNil(t, err) || !assert.Error(t, ErrInvalidContractText, err) {
+			t.FailNow()
```

Comment:
FailNow() (and also the if) is redundant here since it's the last statement anyways